logstash-core/src/test/java/org/logstash/plugins/ConfigurationImplTest.java

```diff
@@ -188,4 +188,26 @@ public void testPasswordDefaultValue() {
         Assert.assertEquals("<password>", p.toString());
     }
 
+    @Test
+    public void testBooleanValues() {
+        PluginConfigSpec<Boolean> booleanConfig = PluginConfigSpec.booleanSetting(booleanKey, false, false, false);
+        Configuration config = new ConfigurationImpl(Collections.singletonMap(booleanKey, "tRuE"));
```

Comment:
First thing that came to mind:

![Spongebob Meme mocking tRuE](https://i.imgflip.com/32xb53.jpg)